portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro

```diff
@@ -40,6 +40,64 @@ definition {
 			value1 = "${pageViewsUsageValue}");
 	}
 
+	macro assertPlanDetails {
```

Comment:
Hi @AfonsoHSantos , it seems the macro ``asserPlanDetails`` is duplicated. Could you please double-check?

Reply:
fixed